v23.1/cockroach-debug-encryption-decrypt.md

```diff
@@ -0,0 +1,64 @@
+---
+title: cockroach debug encryption-decrypt
+summary: Learn the commands for collecting debug information from all nodes in your cluster.
+toc: true
+docs_area: reference.cli
+---
+
+{{site.data.alerts.callout_danger}}
+We strongly recommend only using `cockroach debug encryption-decrypt` when working directly with the [Cockroach Labs support team](support-resources.html).
+{{site.data.alerts.end}}
+
+The [`cockroach debug encryption-decrypt`](cockroach-commands.html) command decrypts store files on a node that are [encrypted at rest](encryption.html) so that they can be examined to add more context to logs gathered from the [`cockroach debug zip`](cockroach-debug-zip.html) command. Unless they are decrypted, Cockroach Labs cannot examine store files that are encrypted at rest.
+
+## Synopsis
+
+~~~ shell
+cockroach debug encryption-decrypt {store_directory} {input_file} {output_file} --enterprise-encryption={encryption_spec} {flags}
+~~~
+
+Replace:
+
+- `{encryption_spec}`: The cluster's [encryption details](encryption.html#starting-a-node-with-encryption).
+- `{store_directory}`: The directory where the node stores data files.
```

Comment:
We usually link `cockroach-start.html#store` for reference. I defer to you on whether that's necessary here.